index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV of Evgeni Borovik</title>
+</head>
+
+<body>
+    <header>
+        <h1>Evgeni Borovik</h1>
+        <hr>
+        <div class="photo">
+            <img src=".\assets\img\photo.jpg" alt="Evgeni Borovik photo">
+        </div>
+    </header>
+    <section class="contacts">
+        <h2 class="header-contacts">Contact information</h2>
+        <p class="paragraph"><em>email:</em> <a href="mailto:anyuser@mail.ru">anyuser@mail.ru</a>
+            <br>
+            <em>skype:</em> eugene_borovik
+            <br>
+            <em>mobile / telegram / viber:</em> <a href="tel:+375296829132">+375 29 682-91-32</a>
+        </p>
+
+    </section>
+    <hr>
+    <section>
+        <h2>Summary</h2>
+        <p class="paragraph">Current position: Strategic Planning Director
```

Comment:
`<p class="paragraph">` class `paragraph` also not very obvious,
I prefer use something like this `<p class="section-text">`